laia/models/htr/laia_crnn.py

```diff
@@ -152,3 +152,18 @@ def get_self_conv_output_size(
         for l in self.conv:
             xs = l.get_batch_output_size(xs)
         return xs
+
+    def get_min_valid_image_size(self, max_search_size: int) -> int:
+        """
+        Compute the minimum image width required to get a valid feature vector of size > 0.
+        :param max_search_size: Max search value for the image width.
+        """
+        for size in range(max_search_size):
+            xs = self.get_self_conv_output_size(torch.tensor([[size, size]]))
+            if torch.count_nonzero(xs) == 2:
+                return size
+        raise ValueError(
+            f"Images of size {max_search_size} pixels "
+            f"would produce invalid output sizes. "
+            f"Please review your model architecture."
+        )
```

Comment:
```suggestion
        raise ValueError(
            dedent(
                f"""
        Images of size {max_search_size} pixels would produce invalid output sizes.
        Please review your model architecture.
        """
            )
        )
```
You should try using [dedent](https://docs.python.org/3/library/textwrap.html#textwrap.dedent). I personally didn't like having three lines.